docs/quickstarts/README.md

```diff
@@ -40,21 +40,21 @@ You can use this simple [preview tool](https://quickstarts-content-preview.surge
 
 
 ## Summary of steps
-This is an overview of the steps you will need to complete to publish an interactive quick start or Learning resource tile in the Hybrid Cloud Console.
+This is an overview of the steps you will need to complete to publish an interactive quick start or Learning resource card in the Hybrid Cloud Console.
 
 1. Write your draft quick start content, and get review and approval from product stakeholders.
 
 2. When the content is ready, create your YAML quick start files in a new directory in the `docs` directory of [**Red Hat Insights quickstarts**](https://github.com/RedHatInsights/quickstarts/tree/main/docs/quickstarts), following the detailed instructions in this `README.md` file. 
 
-3. Get your quick start or Learning resource tile added to the [Hybrid Cloud Console](https://console.redhat.com):
+3. Get your quick start or Learning resource card added to the [Hybrid Cloud Console](https://console.redhat.com):
  
     a. Create a request in [the RHCLOUD Jira project](https://issues.redhat.com/projects/RHCLOUD/issues/) for engineering to merge the content. Add the `platform-experience-services` label to your Jira.
     
     b. Create a pull request (PR) to the `main` branch in the [**Red HatInsights/quickstarts**](https://github.com/RedHatInsights/quickstarts/) repository.
```

Comment:
who should we @ in the actual PR? Also, Martin told me to go into the forum-consoledot-UI Slack channel, and alert _@crc-experience-services-team_ and request that my PR be merged. He said this was due to his reduced bandwith. So, I think we should add that to the README.

Reply:
@Hyperkid123 do we want to mention any specific team members, besides @crc-experience-services-team ? Is it all right to mention our internal Slack channel/tag in this public repo?

Reply:
In the PR you want to tag @RedHatInsights/platform-experience-services as a reviewer, I will setup the Repo so this is done automatically.

Reply:
Thank you for setting that up automatically, @Hyperkid123!